prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -137,6 +137,26 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             .into(Task::class.java)
     }
 
+    /**
+     * Returns true if there is already a record in the report_file table that matches the passed in [senderName],
+     * [senderOrgName], and [digest]
+     */
+    fun checkForDuplicate(
```

Comment:
Minor:  Maybe `isDuplicate` is a more canonical name for this function, since it returns boolean.

Reply:
Updated to `isDuplicateReportFile` to both address this and for more clarity.